src/js/reducers/inspector.js

```diff
@@ -21,7 +21,11 @@ function inspectorReducer(state, action) {
         selected: null
       },
       selected: null,
-      expandedLayers: {}
+      expandedLayers: {},
+      scales: {
+        selected: null,
+        show: false
+      }
```

Comment:
Because selected is just the ID for marks and I didn't want to break existing mark functionality, since there are listeners all over for that. 
Again, this is a little rushed. I figure there will be a clean up of redux once everything gets in there.